go.mod.orig

```diff
@@ -0,0 +1,142 @@
+module github.com/openshift/vmware-vsphere-csi-driver-operator
```

Comment:
Seems like this file was not committed intentionally?

Reply:
Good catch, removed